cfgov/jinja2/v1/_includes/molecules/global-search.html

```diff
@@ -14,8 +14,13 @@
      data-js-hook="behavior_flyout-menu">
     <button class="m-global-search_trigger"
             data-js-hook="behavior_flyout-menu_trigger">
-        <span class="m-global-search_trigger-label">
-            <span class="u-visually-hidden">Search</span>
+        <span class="m-global-search_trigger-open-label">
+            {{ svg_icon('search') }}
+            <span class="u-hide-on-mobile">Search</span>
+        </span>
+        <span class="m-global-search_trigger-close-label">
```

Comment:
It was handled by pseudo elements before (both the icon and the text). I thought it better to wrap the entire thing in a span so that a screen reader can link the icon to the hidden text.